test/e2e/create_test.go

```diff
@@ -108,15 +108,15 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, waitForAnnotations,
 	if expectError {
 		// The time we expect to wait, if the windowsLabel is
 		// not used while creating nodes.
-		creationTime = time.Minute * 5
+		creationTime = time.Minute * 10
```

Comment:
In `testWindowsNodeDeletion` the comment for `testCtx.waitForWindowsNodes()` call says `As per testing, each windows VM is taking roughly 12 minutes to be shown up in the cluster`. This is just plain misleading and should be fixed as part of your PR.

Now in waitForWindowsNodes() , I think you need to do:
```suggestion
		if nodeCount == 0 {
			creationTime = time.Minute * 10
		} else {
			// The time we expect to wait, if the windowsLabel is
			// not used while creating nodes.
			creationTime = time.Minute * 5
		}

```

Reply:
Agreed